ui/src/ducks/app/volumes.js

```diff
@@ -118,8 +118,8 @@ export const updateStorageClassAction = (payload) => {
   return { type: UPDATE_STORAGECLASS, payload };
 };
 
-export const createVolumeAction = (newVolume) => {
-  return { type: CREATE_VOLUMES, payload: { newVolume } };
+export const createVolumeAction = (newVolumes) => {
```

Comment:
I would rename this to `createVolumesAction` to reflect the batch creation possibility